Makefile

```diff
@@ -128,7 +128,7 @@ cluster/seed/managed/postgressnapshot:
 
 .PHONY: cluster/clean
 cluster/clean:
-	@$(KUSTOMIZE) build config/crd | kubectl delete -f -
+	@$(KUSTOMIZE) build config/crd | oc delete -f -
```

Comment:
any reason to use oc over kubectl here?

Reply:
@laurafitzgerald  I didn't have `kubectl` locally : ), so i found that it was the only place where `oc` was not used but kubectl. But , as I see now - Makefile changed, and maybe need rebase ... No already `kubectl` anyway in Makefile

Reply:
sorry, ignore please my reply, regarding rebase & missing kubectl. I looked at not correct Makefile.
So answer -  I didn't have kubectl locally : ), and this is the only place where kubectl was used, so didn't see any reason to use it

Reply:
ok. no problem. I think we had agreed a long time ago that we would use kubectl in all cases where possible and only oc where we had to but looks like it's used elsewhere so I think this is fine.